tests/test/io/pedestal/http/route_repl_test.clj

```diff
@@ -45,23 +45,64 @@
         alt-routes #{["/bye" :get #'bye-handler :route-name ::bye]}]
     (is (fn? f))
 
-    (is (= (simplify (route/expand-routes sample-routes))
-           (simplify (f))))
+    (let [out-str (with-out-str
```

Comment:
I decided to link the printing of the routing table directly to generated _function_ so that it only occurs when uring `route-from`. There _might_ be cases of production applications that still use a function to provide the specification.